doc/api/api_changes_3.3/behaviour.rst

```diff
@@ -155,3 +155,12 @@ support for it will be dropped in a future Matplotlib release.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Previously, keyword arguments were silently ignored when no positional
 arguments were given.
+
+
+`.get_minorticklabels` and `.get_majorticklabels` now returns plain list
```

Comment:
```suggestion
`.get_minorticklabels` and `.get_majorticklabels` now returns plain list
```
Either also qualify by `.Axis`, or make them literal. Don't forget to adapt the length of the header undline :smile:.